packages/wrangler/src/index.tsx

```diff
@@ -347,6 +350,23 @@ export async function main(argv: string[]): Promise<void> {
                 "utf-8"
               )
             );
+
+            await writeFile(
+              pathToPackageJson,
+              JSON.stringify(
+                {
+                  ...packageJsonContent,
+                  scripts: {
+                    ...packageJsonContent.scripts,
+                    start: "wrangler dev src/index.ts",
```

Comment:
Thinking about it, I don't think we should overwrite this, it would also overwrite any existing `start`/`deploy` scripts. So my suggestion - if `start` or `deploy` already exist, then we don't write either at all. At the very end of this command
- If start/deploy didn't exist in the `package.json`, and we put in these commands instead, we log a message to the effect of "To start developing on your worker, run `npm start`. To publish your worker on to the internet, run `npm run deploy`". 
- If we _didn't_ write these commands to package.json because they already existed, then we log the same message, but with `npx wrangler dev src/index.ts`/`npx wrangler publish src/index.ts`.

Reply:
Just want to confirm, if the user chose to not create a `package.json` we don't want to override that decision creating these scripts as well?

Reply:
I'd think so, yeah.